.github/CODEOWNERS

```diff
@@ -105,6 +105,9 @@
 /mlir/**/*ToSPIRV/ @antiagainst @kuhar
 /mlir/tools/mlir-tblgen/SPIRVUtilsGen.cpp @antiagainst @kuhar
 
+# WASM Dialect in MLIR.
```

Comment:
I'm seeing errors here:
```
Unknown owner on line 109: make sure @flemairen6 exists and has write access to the repository
Unknown owner on line 109: make sure @lforg37 exists and has write access to the repository
```

Reply:
They're not committers in LLVM I think, so Github complains because these files are Github specific. I'd leave that out for this PR and wait for them to be added as committers to reintroduce that change.

Reply:
Got it, reverted for the moment